cdap-docs/developer-manual/source/building-blocks/transaction-system.rst

```diff
@@ -143,15 +143,8 @@ By default, CDAP will start an implicit transaction for these methods:
   programs and sub-programs (flowlets, service handlers, and workflow actions), with the
   exception of worker programs.
 
-For example, as shown in the :ref:`WordCount example <examples-word-count>`, the
-``RetrieveCountsHandler`` uses the *wordCounts* ``KeyValueTable``. CDAP implicitly starts a
-transaction for this handler method, and the handler can rely on the transactional
-consistency of the data it reads from the dataset:
-
-.. literalinclude:: /../../../cdap-examples/WordCount/src/main/java/io/cdap/cdap/examples/wordcount/RetrieveCountsHandler.java
-   :language: java
-   :lines: 118-140
-   :dedent: 2
+CDAP implicitly starts a transaction for this handler method, and the handler can rely
+on the transactional consistency of the data it reads from the dataset.
```

Comment:
Is this still correct? Or did we change to make all transactions explicit in 6.0? @chtyim might know